src/cli/bootstrap/bootstrap.ts

```diff
@@ -180,6 +187,21 @@ export default async function bootstrap(args: string[]) {
             },
             EosioUtil.createSigner(newPrivateKey)
         );
+        await idContract.setAccountType('id.tmy', AccountTypeEnum.App, signer);
+        await idContract.setAccountType('eosio', AccountTypeEnum.App, signer);
+        await idContract.setAccountType('onocoin.tmy', AccountTypeEnum.App, signer);
+
+        //call resource params
+        await eosioContract.setresparams(0.000005769230769, 8 * 1024 * 1024 * 1024, 0.25, signer);
+        //call resource params
```

Comment:
better to define These before we call `setresparams()` and then use it during the function call